### PR TITLE
Allow change the issuer

### PR DESCRIPTION
**Pull request by avdata99**
This could be required.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -117,7 +117,10 @@ Optional::
      # Default: urn:oasis:names:tc:SAML:2.0:nameid-format:persistent
      ckanext.saml2auth.sp.name_id_format = urn:oasis:names:tc:SAML:2.0:nameid-format:persistent urn:oasis:names:tc:SAML:2.0:nameid-format:transient
 
-
+    # Issuer
```
[the remark this answers is not in the document]
Comment (mbocevski) [reply README.rst:120]: Yes, please use entity_id instead of issuer.

File: ckanext/saml2auth/spconfig.py [hunk 2]
```diff
@@ -6,6 +6,10 @@
 
 
 def get_config():
+    """ Get the config
+        Read: https://pysaml2.readthedocs.io/en/latest/howto/config.html
+        """
+
```
Comment (mbocevski) [inline ckanext/saml2auth/spconfig.py:12]: Remove whitespace. Make sure you run pep8 checks before pushing.

**Review by mbocevski — COMMENTED** [review]
Looks good, minor changes needed.

File: ckanext/saml2auth/spconfig.py [hunk 3]
```diff
@@ -23,8 +27,16 @@ def get_config():
...
         ckan_config.get(u'ckanext.saml2auth.idp_metadata.remote_cert')
 
+    issuer = ckan_config.get(u'ckanext.saml2auth.issuer', u'urn:mace:umu.se:saml:ckan:sp')
+    response_signed = asbool(ckan_config.get(u'ckanext.saml2auth.want_response_signed', True))
+    assertion_signed = asbool(ckan_config.get(u'ckanext.saml2auth.want_assertions_signed', False))
+    any_signed = asbool(ckan_config.get(u'ckanext.saml2auth.want_assertions_or_response_signed', False))
+    key_file = ckan_config.get(u'ckanext.saml2auth.key_file_path', None)
+    cert_file = ckan_config.get(u'ckanext.saml2auth.cert_file_path', None)
+    attribute_map_dir = ckan_config.get(u'ckanext.saml2auth.attribute_map_dir', None)
+
```
Comment (duskobogdanovski) [inline ckanext/saml2auth/spconfig.py:37]: Please document all additional fields in the README.rst and add tests.

File: README.rst (hunk 1 above)
[the remark this answers is not in the document]
Comment (avdata99) [reply README.rst]: Done @adborden @mbocevski

File: ckanext/saml2auth/spconfig.py (hunk 2 above)
Comment (avdata99) [reply ckanext/saml2auth/spconfig.py]: Done @mbocevski

File: ckanext/saml2auth/spconfig.py (hunk 3 above)
Comment (avdata99) [reply ckanext/saml2auth/spconfig.py:37]: Done @duskobogdanovski

**Review by duskobogdanovski — APPROVED** [review]
LGTM

File: ckanext/saml2auth/spconfig.py (hunk 2 above)
Comment (adborden) [reply ckanext/saml2auth/spconfig.py]: @mbocevski can we enable pep8 checks as part of CI and then enforced with GH protected branches? That way you can focus your review on the important stuff and leave things like whitespace checks to the computer :)

Comment (mbocevski) [reply ckanext/saml2auth/spconfig.py]: > can we enable pep8 checks as part of CI

@adborden it already is https://github.com/keitaroinc/ckanext-saml2auth/blob/main/bin/travis-run.sh#L5 :) The travis CI checks failed, I was just pointing it out as part of the review :) Happy new year!

**Pull request merged**